### DISCUSSION TITLE
Docs on compiling .deb packages locally with yarn package cmd?

### DISCUSSION THREAD

**Discussion by ghost** [coder/code-server#2739] [Q&A]
I made local change and want to install a development `code-server` release in `/usr/local/bin/code-server`. 

Building `.deb` seems the fastest way to do this but `yarn package` gives this error:

```
/tmp/code-server# yarn package
yarn run v1.22.5
$ ./ci/build/build-packages.sh
done (release-packages/code-server-3.9.0-linux-amd64)
./ci/build/build-packages.sh: line 43: envsubst: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

1. Is there docs on building packages or installing to `/usr/local/bin/code-server` locally from source?
2. Is `code-server/out/node/entry.js` the same as the compiled `/usr/local/bin/code-server`?

**Comment by oxy** [comment]
I'm not entirely sure what the problem is - but generally speaking I wouldn't recommend installing experimental versions of `code-server` to the whole system for testing: I would suggested either using `yarn watch` or waiting for the compile to finish successfully and then running `node out/node/entry.js` if `yarn watch` is using too much memory.

**Comment by RobertCNelson** [comment]
@bachmannn envsubst comes from the gettext-base package:

https://github.com/cdr/code-server/blob/v3.9.0/ci/images/debian10/Dockerfile#L23-L24

You'll also need the go scripts here:

https://github.com/cdr/code-server/blob/v3.9.0/ci/images/debian10/Dockerfile#L43-L46

Regards,
